ts/sdk/src/types/types.ts

```diff
@@ -159,9 +160,26 @@ export type CompetitionRoundWinnerRecord = {
 	ts: BN;
 };
 
+export type CompetitorSettledRecord = {
+    roundNumber: BN;            // count of rounds for this competition
+    competitor: PublicKey;           // public key of corresponding competitor account
+    competition: PublicKey;          // public key of corresponding competition account
+    competitorAuthority: PublicKey; // public key of authority of competitior
+    status: CompetitorStatus, // status of whether the competitior is in good standing
+    unclaimedWinnings: BN; // competitors current unclaimed winnings (they won't be considered for this draw if non-zero)
```

Comment:
> they won't be considered for this draw if non-zero

this part is a little vague, does this mean the competitor won't be considered for the settling draw if they have unclaimed winnings?

Reply:
yes. if you already won and havent claimed, the competition tries to let someone else win

Reply:
just for extra context these comments are downstream from the events.rs file btw @ChesterSim